sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager+TransferDelegate.swift

```diff
@@ -78,7 +79,10 @@ extension URLSessionTransferManager: TransferDelegate {
             for restorationId in Set(restorationIds) {
                 guard let delegate = self.client(forRestorationId: restorationId) as? TransferDelegate else { continue }
                 let transfersForId = transfers.filter { ($0 as? TransferImpl)?.clientRestorationId == restorationId }
-                delegate.transfersDidUpdate(transfersForId)
+                for transfer in transfersForId {
+                    let progress = (transfer as? BlobTransfer)?.progress
+                    delegate.transfer(transfer, didUpdateWithState: transfer.state, andProgress: progress)
```

Comment:
Need to also call the progressHandler here?

Reply:
Done